docs/how-it-works.md

```diff
@@ -36,7 +36,7 @@ The `.api/` directory is wher the CLI installation process stores all of its inf
     ├── readme/
     |   ├── node_modules/
     │   ├── index.js
-    │   ├── index.d.ts   // All types for the SDK, ready to use in your an IDE.
+    │   ├── index.d.ts   // All types for the SDK, ready to use in your IDE.
```

Comment:
I might have this same typo in the storage library

Reply:
Did a quick search and couldn't find anything!